modules/apps/portal-language/portal-language-lang/src/main/resources/content/Language.properties

```diff
@@ -14738,6 +14738,7 @@ search-with-index-key-description=Leave this checked to use the index to search
 search-with-query-string-syntax=Search with Query String Syntax
 search-with-query-string-syntax-description=Use Query String query syntax to match one or more fields with a single operator (OR or AND).
 search-x=Search {0}
+search-x-in-x=Search "{0}" in {1}
```

Comment:
Why does {0} need the quotation marks here?

Reply:
For consistency, the original string had quotation marks around the queryValue

Reply:
Thank you for clarifying.